src/modules/addObj.js

```diff
@@ -0,0 +1,17 @@
+import storage from './storage.js';
+import mainArr from './const.js';
+import addHtml from './addHtml.js';
```

Comment:
I must say that here, you have made good practice by importing functions between JS files inside the same module